auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -167,21 +155,6 @@ public void shouldInitWithContext() {
         assertNotNull(WebAuthProvider.getManagerInstance());
     }
 
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldNotResumeLoginWithRequestCodeWhenNotInit() {
```

Comment:
removed as were dupe. Search for `assertFalse(WebAuthProvider.resume(` and you will find two consecutive tests for this same scenario. They were missing the "reset state" call at the beginning as well, which I added